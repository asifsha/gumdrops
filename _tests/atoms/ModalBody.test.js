/* globals mount */
import React from 'react';
import ModalBody from '../../components/atoms/ModalBody';

const defaultProps = {
    className: 'modal-class',
    style: { width: '100px' }
};

describe('Expect <ModalBody>', () => {
    it('to render', () => {
        const wrapper = mount(
            <ModalBody {...defaultProps}>
                <div>My modal content</div>
            </ModalBody>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
