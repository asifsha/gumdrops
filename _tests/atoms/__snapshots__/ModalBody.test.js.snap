// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <ModalBody> to render 1`] = `
<ModalBody
  className="modal-class"
  style={
    Object {
      "width": "100px",
    }
  }
>
  <div
    className="gds-modal__body modal-class"
    style={
      Object {
        "width": "100px",
      }
    }
  >
    <div>
      My modal content
    </div>
  </div>
</ModalBody>
`;
